template/content/app.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var fetchConfig = require('zero-config');
+
+var ApplicationClients = require('./clients.js');
+
+module.exports = Application;
+
+function Application(options) {
+    if (!(this instanceof Application)) {
+        return new Application(options);
+    }
+
+    var self = this;
+    options = options || {};
+
+    self.config = fetchConfig(__dirname, {
+        dcValue: 'todo',
+        seed: options.seedConfig,
+        loose: false
+    });
+
+    self.clients = ApplicationClients(self.config, {
+        logger: options.logger,
+        statsd: options.statsd
+    });
+
+    var channel = self.clients.appChannel;
+    var thrift = self.clients.tchannelThrift;
+
+    thrift.register(
+        channel, 'MyService::health_v1', self, Application.health
+    );
+
+    // TODO remove example endpoints
+    thrift.register(
+        channel, 'MyService::get_v1', self, Application.get
+    );
+    thrift.register(
+        channel, 'MyService::put_v1', self, Application.put
+    );
+
+    // Example data structure on application
+    self.exampleDb = {};
+}
+
+Application.prototype.bootstrap = function bootstrap(cb) {
+    var self = this;
+
+    self.clients.bootstrap(cb);
+};
+
+Application.prototype.destroy = function destroy() {
+    var self = this;
+
+    self.clients.destroy();
+};
+
+Application.health = function health(app, req, head, body, cb) {
```

Comment:
I previously had an `endpoints.js` which didn't feel quite right. In autobahn i have an endpoints folder but that feels quite heavy for simple services.

I think this might actually be the right balance; your endpoints should be small and you will only have a couple of them; put them in here and if they get too big move the code out of the endpoints function.